jest.config.v2.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
       },
       reporters: ["default", "jest-junit"],
       setupFilesAfterEnv: ["<rootDir>/src/v2/jest.envSetup.ts"],
+      testEnvironment: "jest-environment-jsdom-thirteen",
```

Comment:
TODO: add context for this change

Reply:
[Added](https://github.com/artsy/force/pull/7629/commits/ec911988e865f22a0f6bb03729e1bde61f59d496#diff-8aa60dd701d62e5679b59339faf9418a8b5764591cca2779bdcf5998884303e3) to Hacks.md

Reply:
Thanks for adding to hacks 👍 Since its easy, we still might want to add a `// comment` here in the file too